runtime/j9vm/j9vmnatives.xml

```diff
@@ -25,6 +25,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<export name="JNI_CreateJavaVM" />
 		<export name="JNI_GetCreatedJavaVMs" />
 		<export name="JNI_GetDefaultJavaVMInitArgs" />
+		<export name="JITServer_CreateServer" />
```

Comment:
This should not be necessary (see comment above).

Reply:
I think we do need to export `JITServer_CreateServer`, the others are not required as they are accessed as function pointers. `JITServer_CreateServer` can be called directly by the user.